hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenCreateHandler.java

```diff
@@ -231,13 +229,16 @@ private Token buildToken(
     private ExpiryMeta getExpiryMeta(final long consensusTime, @NonNull final TokenCreateTransactionBody op) {
         final var impliedExpiry =
                 consensusTime + op.autoRenewPeriodOrElse(Duration.DEFAULT).seconds();
+        final var accountId = AccountID.newBuilder()
+                .shardNum(op.hasAutoRenewAccount() ? op.autoRenewAccount().shardNum() : NA)
+                .realmNum(op.hasAutoRenewAccount() ? op.autoRenewAccount().realmNum() : NA)
+                .accountNum(op.hasAutoRenewAccount() ? op.autoRenewAccount().accountNumOrElse(NA) : NA)
+                .build();
```

Comment:
same here as above. no need to create accountId

Reply:
Updated.